src/pubtools/sign/results/signing_results.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+import dataclasses
+
+from typing import ForwardRef
+
+from pubtools.sign.operations.base import SignOperation
+from pubtools.sign.results import SignerResults
+from pubtools.sign.results.sign_results import SignResults
+
+
+Signer = ForwardRef("Signer")
+
+
+@dataclasses.dataclass
+class SigningResults:
```

Comment:
Could you rethink the naming of these classes? You have sign result, signing result, and signer result. Its confusing.

Reply:
would it be enough if I rename sign results to operation results?

Reply:
I guess